### PR TITLE
MTA-556 Docs Improvement - Custom Migration Targets (RuleSets)

### PR DESCRIPTION
**Pull request by sbeskin-redhat**
Resolves: https://issues.redhat.com/browse/MTA-556
MTA 6.1.0+
Preview: https://deploy-preview-704--windup-documentation.netlify.app/docs/web-console-guide/master/index.html#mta-web-creating-custom-migration-targets_user-interface-guide

### CODE REVIEW

**Comment by sbeskin-redhat** [comment]
@rromannissen 
Please review this PR.

File: docs/topics/mta-web-creating-custom-migration-targets.adoc [hunk 1]
```diff
@@ -6,7 +6,9 @@
 [id="mta-web-creating-custom-migration-targets_{context}"]
 = Creating custom migration targets
 
-Users with `admin` permissions can create and maintain custom rulesets (migration targets). Non-admin users then can use those migration targets for analyzing applications.
+The purpose of this feature is to enable architects or users with `admin` permissions to create and maintain custom rulesets (migration targets). Architects can upload custom rule files and assign them to various custom migration targets. The custom migration targets are then available for selecting in the analysis configuration wizard.
```
Comment (rromannissen) [inline docs/topics/mta-web-creating-custom-migration-targets.adoc:9]: "The purpose of this feature is to enable architects or users with `admin` permissions to create and maintain custom rulesets associated to custom migration targets"

**Review by rromannissen — CHANGES_REQUESTED** [review]
Only one small nitpick, the rest looks good to me!

File: docs/topics/mta-web-creating-custom-migration-targets.adoc (hunk 1 above)
Comment (sbeskin-redhat) [reply docs/topics/mta-web-creating-custom-migration-targets.adoc]: @rromannissen 
Changed, please see.

File: docs/topics/mta-web-creating-custom-migration-targets.adoc [hunk 2]
```diff
@@ -6,7 +6,9 @@
 [id="mta-web-creating-custom-migration-targets_{context}"]
 = Creating custom migration targets
 
-Users with `admin` permissions can create and maintain custom rulesets (migration targets). Non-admin users then can use those migration targets for analyzing applications.
+The purpose of this feature is to enable architects or users with `admin` permissions to create and maintain custom rulesets associated with custom migration targets. Architects can upload custom rule files and assign them to various custom migration targets. The custom migration targets are then available for selecting in the analysis configuration wizard.
+
+Using ready-made custom migration targets makes the cumbersome task of configuring custom rules for each analysis run unnecessary. This simplifies analysis configuration and execution for non-admin users or third-party developers and makes using the {ProductShortName} tool easier and more straightforward.
```
Comment (RichardHoch) [inline docs/topics/mta-web-creating-custom-migration-targets.adoc:12]: Suggested rewrite: If you are an architect or a user with `admin` privileges, you can create and maintain custom rulesets associated with custom migration targets.  You can upload custom rule files ...

Using ready-made custom migration targets eliminates the cumbersome task of configuring custom rules for each analysis run. This simplifies analysis configuration and execution for non-admin users or third-party developers and makes using the {ProductShortName} easier and more straightforward.

**Comment by RichardHoch** [comment]
LGTM -- I suggested a couple small tweaks, but up to you.

**Pull request merged**